app/Resources/views/participant_history/index.html.twig

```diff
@@ -169,6 +170,16 @@
                                             <button type="submit" class="button tiny alert"><i class="fa fa-trash"></i>
                                             </button>
                                         </form>
+                                </td>
+                                <td>
+                                        <form
+                                                action="{{ path('assistant_history_edit', {id: ah.id}) }}"
+                                                method="GET"
+                                        >
+                                            <button type="submit" class="button tiny"><i class="fa fa-pencil"></i>
+                                            </button>
+
+                                        </form>
```

Comment:
Burde dette heller være en `<a>`, siden den denne knappen kun skal linke til skjemaet for å redigere assistenthistorien?